src/components/Answer.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button } from "@chakra-ui/react";
 import { MouseEventHandler } from "react";
 
-export default function Answer(props: { id: string; background: string; text: string, setAnswer: MouseEventHandler }) {
+export default function Answer(props: { id: string; background: string; text: string | undefined, setAnswer: React.MouseEventHandler<HTMLButtonElement> }) {
```

Comment:
When can `text` be `undefined`? If it's for the scenario when it's waiting for a question to be loaded from the backend, it should display a waiting animation.

Reply:
Good idea, I will try that